website/src/logger.ts

```diff
@@ -0,0 +1,45 @@
+import winston, { type Logger } from 'winston';
+import DailyRotateFile from 'winston-daily-rotate-file';
+
+import { type InstanceLogger } from './types/logMessage.ts';
+
+let _logger: Logger | undefined;
+
+const getLogger = (): Logger => {
+    if (_logger === undefined) {
+        const logPath = import.meta.env.LOG_DIR;
+        const serverLogFile = `${logPath}/website.log`;
```

Comment:
```suggestion
        const serverLogFile = `${logPath}/dashboards_website.log`;
```
We have also other websites on the server. Just to make it more cleare where the log comes from.
Or we find a name for the backend and the website ;)